reactioncommerce/reaction-admin
# Use GraphQL for e-mail template table and update form

Resolves #227
Impact: **major**
Type: **performance|refactor|chore**

## Issue
The current e-mail template table and update form use the `Templates` Meteor pub/sub to pull data, and a Meteor method to update templates. This is outdated and should be replaced by a GraphQL query and mutation.

## Solution
Refactor the e-mail template table and update form to use GraphQL instead.

## Breaking changes
None.

## Testing
Test with https://github.com/reactioncommerce/api-plugin-translations/pull/15.

1. Go to `Settings > Email Templates`.
2. Make sure your e-mail templates display in the table.
3. Click a row of the table and edit the e-mail template.
4. Make sure that the update works.
